src/Modules/CalcOffence.lua

```diff
@@ -1818,6 +1902,12 @@ function calcs.offence(env, actor, activeSkill)
 			output.HitTime = output.Time * skillData.hitTimeMultiplier
 			output.HitSpeed = 1 / output.HitTime
 		end
+		
+		-- Other Misc DPS multipliers (like custom source)
+		skillData.dpsMultiplier = ( skillData.dpsMultiplier or 1 ) * ( 1 + skillModList:Sum("INC", cfg, "DPS") / 100 ) * skillModList:More(cfg, "DPS")
+		if env.configInput.repeatMode == "FINAL" or skillModList:Flag(nil, "OnlyFinalRepeat") then
+			skillData.dpsMultiplier = skillData.dpsMultiplier / (output.Repeats or 1)
+		end
```

Comment:
```suggestion
		if env.configInput.repeatMode == "FINAL" or skillModList:Flag(nil, "OnlyFinalRepeat") then
			skillData.dpsMultiplier = skillData.dpsMultiplier / (output.Repeats or 1)
		end
```
This could probably be implemented using hitTimeMultipler which would probably make more sense then a dps multiplier
e.g.
```lua

		if env.configInput.repeatMode == "FINAL" or skillModList:Flag(nil, "OnlyFinalRepeat") then
			skillData.hitTimeMultiplier = output.Repeats or 1
		end
		if skillData.hitTimeOverride and not skillData.triggeredOnDeath then
			output.HitTime = skillData.hitTimeOverride
			output.HitSpeed = 1 / output.HitTime
			--Brands always have hitTimeOverride
			if skillCfg.skillName and skillCfg.skillName:match("Brand") then
				output.BrandTicks = m_floor(output.Duration * output.HitSpeed)
			end
		elseif skillData.hitTimeMultiplier and output.Time and not skillData.triggeredOnDeath then
			output.HitTime = output.Time * skillData.hitTimeMultiplier
			output.HitSpeed = 1 / output.HitTime
		end
		
		-- Other Misc DPS multipliers (like custom source)
		skillData.dpsMultiplier = ( skillData.dpsMultiplier or 1 ) * ( 1 + skillModList:Sum("INC", cfg, "DPS") / 100 ) * skillModList:More(cfg, "DPS")
	end
```

Reply:
This does make more sense to me unless there are issues with it.